task/build-image-manifest/0.1/build-image-manifest.yaml

```diff
@@ -46,7 +46,7 @@ spec:
     - name: TLSVERIFY
       value: $(params.TLSVERIFY)
   steps:
-  - image: quay.io/redhat-appstudio/buildah:v1.31.0@sha256:34f12c7b72ec2c28f1ded0c494b428df4791c909f1f174dd21b8ed6a57cf5ddb
+  - image: quay.io/redhat-appstudio/multi-platform-runner:01c7670e81d5120347cf0ad13372742489985e5f
```

Comment:
What is the reason for this change?

Reply:
oops, this was originally because I was going to use podman instead, which is part of that image, but it turned out that the issue was not with buildah, but rather with the use of tag + digest. I will revert.